node/src/components/consensus/era_supervisor.rs

```diff
@@ -67,8 +67,12 @@ pub use self::era::Era;
 /// fault tolerance threshold.
 const FTT_EXCEEDED_SHUTDOWN_DELAY_MILLIS: u64 = 60 * 1000;
 
+/// The number of eras across which evidence can be cited.
+/// If this is 1, you can cite evidence from the previous era, but not the one before that.
+/// To be able to detect that evidence, we also keep that number of full past eras in memory.
+const PAST_EVIDENCE_ERAS: u64 = 1;
 /// The number of active eras that are kept in memory in addition to the current one.
-const PAST_ACTIVE_ERAS: u64 = 2;
+const PAST_ACTIVE_ERAS: u64 = 2 * PAST_EVIDENCE_ERAS;
```

Comment:
Is it really related?

Reply:
I think so, yes:
We want to keep the past `PAST_EVIDENCE_ERAS` around as _full_ eras so that we can detect all the equivocations we would be allowed to cite.
Then, we have to keep a further `PAST_EVIDENCE_ERAS` in evidence-only mode because the earliest full era could cite evidence from them.